features/admin.extensions.v1/components/administrators/pages/administrators.tsx

```diff
@@ -261,6 +262,12 @@ const CollaboratorsPage: FunctionComponent<CollaboratorsPageInterface> = (
         administratorConfig.enableAdminInvite
     );
 
+    const { data: consoleApplicationFilter } = useApplicationList(null, null, null, "name eq Console");
+
+    const consoleId: string = useMemo(() => {
+        return consoleApplicationFilter?.applications[0]?.id;
```

Comment:
remove the unecessary `return` keyword.

Reply:
Improved with 7da71001ffb8fb165c506dd33ee01229f6737a8f